problems/5.loops.js

```diff
@@ -1,33 +1,51 @@
 /** Returns every even number from 0 to a given number
-* @param {number} num - The number to iterate up to
-* @param {number[]} - An array of even numbers from 0 to {num}
-*/
-function evenNumbers() {
-
+ * @param {number} num - The number to iterate up to
+ * @param {number[]} - An array of even numbers from 0 to {num}
+ */
+function evenNumbers(num) {
+  let newArray = [];
+  for (i = 0; i <= num; i++) {
+    if (i % 2 === 0) {
+      newArray.push(i);
+    }
+  }
+  return newArray;
 }
 
 /** Returns all multiples of a given number up to 1000
  * Example if num was 3 the output would be [0, 3, 6, 9, 12, ..., 999]
-* @param {number} num - The number to find multiples of
-* @returns {number[]} - An array of all multiples of {num} up to 1000
-*/
-function multiplesToOneThousand() {
-
+ * @param {number} num - The number to find multiples of
+ * @returns {number[]} - An array of all multiples of {num} up to 1000
+ */
+function multiplesToOneThousand(num) {
+  let newMultiple = 0;
+  let newArray = [];
+  for (i = 0; i <= 1000; i++) {
+    newMultiple = num * i;
```

Comment:
Make sure that you are finding the multiples of a given number as opposed to just multiplying the given number.